exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,16 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+function checkRole (name , danielsRole) {
+  if( danielsRole === "mentor"){
+    return "Hi, I'm Daniel, I'm a mentor."
+  } else if (danielsRole="student"){
+    "Hi, I'm Daniel, I'm a student."
+  }
+}
```

Comment:
nice use of strict equality for the first if statement, you're just missing it for the second statement (you just have a single =)